src/util-base64.c

```diff
@@ -235,7 +235,7 @@ static inline Base64Ecode DecodeBase64RFC4648(uint8_t *dest, uint32_t dest_size,
      * violate the destination buffer size */
     if (bbidx > 0) {
         padding = bbidx > 1 ? B64_BLOCK - bbidx : 2;
-        uint32_t numDecoded_blk = ASCII_BLOCK - (padding < B64_BLOCK ? padding : ASCII_BLOCK);
```

Comment:
is there way to make this easier to read? It's hard to track the logic of bbidx, padding, B64_BLOCK, ASCII_BLOCK here

Reply:
I could add proper comments and maybe make it an if else statement wherever this logic is in place.